operations/network/management_network/bican_technical_details.md

```diff
@@ -0,0 +1,252 @@
+- UAI addressing and routing over the HSN interfaces for the NCN workers is required.
+- API endpoints in MetalLB for the CHN will be accessible over NCN worker HSN interfaces (via ECMP Layer 3 routing).
+  - MetalLB will peer with the Edge Routers to supply load balanced API access.
+
+#### 2.5.2 When configuration occurs
+
+- Installation as part of a virtual routing instance on the Edge Routers.
+
+#### 2.5.3 Ability to change post install
+
+- NOT RECOMMENDED - Edge Router controls external access so change scope is limited.
+- NO - Node images could be changed but routing and IP changes to CFS images would need extensive testing to certify.
+
+## 3 Management CAN(CMN)
+
+The original CAN released in Shasta 1.1 contained the ability to access NCN workers, masters and storage directly via SSH for administrative purposes.
+This administrative traffic was co-mingled with general user traffic for jobs.
+Based on customer requests a new mechanism for administrative access to workers, masters and storage nodes will be added in CSM 1.2.
+The new Customer Management Network (CMN) will be created as **a separate-and-distinct VLAN and Subnet on the Management Network** and uplink at the edge to the Customer network.
+This new CMN network will allow SSH into the NCNs and CAN access will be disallowed.
+NOTE this is generally for ingress access for administrative purposes.
+
+### 3.1 Traffic Separation and Routing
+
+Enabling the CMN at installation time will have the following effects:
+
+- Adding uplinks to the Customer Site similar to the original CAN.
+- Creating of a new VLAN on the Management Network.
+- Adding a new subnet and routing to the Management Network and NCNs.
+
+### 3.2 Endpoint Naming
+
+If enabled during installation a ".CMN" suffix will be generated by CSI and used in SLS and Authoritative DNS.
+The "plumbing" of this will occur as previously described in the CHN.
+
+Examples:
+
+- SSH `ncn-w001.cmn.tld`
+
+### 3.3 Endpoint Addressing
+
+For the CSM 1.2 release the CMN will only be available via customer-supplied IPv4 addressing.
+
+### 3.4 Changes
+
+#### 3.4.1 Touchpoints: effects and changes
+
+- CMN required beginning with this release.
+  - Customer will supply a subnet similar to the way the CAN is deployed.
+  Sizing is Number of NCNs plus a couple more addresses (TBD).
+- Edge access to the CMN will need to be configured with the customer site.
+  - ACL development.
+- The Management network will require the following changes:
+  - Addition of the new CMN VLAN.  This should be similar to the existing CAN configuration.
+  - Termination of the new CMN VLAN on ports supporting NCNs.
+  - Addition of Customer-supplied CMN IP&#39;s to the management switches to support routing.
+- NCN workers, masters and storage will require the following changes:
+  - Image support for CMN VLAN, addressing and routing.
+- CSI changes to support the new network and naming.
```

Comment:
Does this mean changes to the CSI inputs a customer provides? If it's just changes we had to make to CSI internally, probably we don't need to mention it in the customer-facing docs.